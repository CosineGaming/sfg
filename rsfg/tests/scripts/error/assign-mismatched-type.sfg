fn main()
	var x = 5
	x = "five"

