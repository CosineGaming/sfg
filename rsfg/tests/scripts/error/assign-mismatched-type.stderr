[ERROR] mismatched type, expected int at 3:2, got str at 3:6